test/fixtures/served/iframe.html

```diff
@@ -34,14 +40,15 @@
             width: event.data.width,
           }, '*');
     } else if (event.data.type == 'subscribeToEmbedState') {
+      var sentinel;
       if (event.data.is3p) {
         if (event.data.amp3pSentinel) {
```

Comment:
Can we make this a [conditional expression too](https://github.com/ampproject/amphtml/pull/6922/files/bf60e8429af2c92da2d858dc7a1a6890e49da167..190e6ff0b69eb7a9cf84e10de50b268eb3a45e29#r95856378)?